src/androidTest/java/com/couchbase/lite/DatabaseAttachmentTest.java

```diff
@@ -931,7 +956,7 @@ public void testGzippedAttachmentByBase64() throws Exception {
         assertTrue(Arrays.equals(bytes, receivedBytes));
     }
 
-    private RevisionInternal putDocWithAttachment(String docID, String attachmentText, boolean compress) throws CouchbaseLiteException {
+    public RevisionInternal putDocWithAttachment(String docID, String attachmentText, boolean compress) throws CouchbaseLiteException {
```

Comment:
Initially I tried to use this method from other test class. As I did not do so, It should be stay `private`.